assets/admin_themes/Sea_Green/admin_customizations.scss

```diff
@@ -192,3 +192,9 @@ blockquote {
 #surveySettingsForThisGroup {
   column-gap: 20px;
 }
+
+#usermanagement--identity-gridPanel tr.disabled {
+  color            : $g-400;
+  pointer-events   : auto;
+  cursor           : not-allowed;
+}
```

Comment:
did you check if this needs to also be adjusted in other themes? same with the next usermanagement.js file